src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -37,6 +37,7 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
   EnrollStatus: typeof EnrollStatus = EnrollStatus;
   courseId: string = '';
   coursePresent?: boolean;
+  isLoading?: boolean;
```

Comment:
I think there is no need to mark it as optional. It can be false at the begining